operations/update.go

```diff
@@ -85,12 +85,21 @@ func Update() cli.Command {
 					return errors.Errorf("Failed to get installation path: %s", err)
 				}
 
-				grip.Infoln("Unlinking existing binary at", binaryDest)
-				err = syscall.Unlink(binaryDest)
-				if err != nil {
-					return err
+				winTempFileBase := strings.TrimSuffix(filepath.Base(binaryDest), ".exe")
+				winTempDest := filepath.Join(filepath.Dir(binaryDest), winTempFileBase+"-old.exe")
+				if runtime.GOOS == "windows" {
+					grip.Infoln("Moving existing binary", binaryDest, "to", winTempDest)
+					if err = os.Rename(binaryDest, winTempDest); err != nil {
```

Comment:
The `cmd` at the end is supposed to clean it up. The deletion has to occur at the end because it fails in between rename and the final deletion, it would delete both the new and old binaries and leave you with no binary at all. By doing it this way, it will most likely will leave you with just the new binary (although there's some chance it could leave both a new and old binary behind if `cmd` fails for whatever reason).